curriculum/challenges/english/14-responsive-web-design-22/learn-css-grid-by-building-a-magazine/6143a778bffc206ac6b1dbe3.md

```diff
@@ -123,8 +118,9 @@ assert(document.querySelectorAll('.text p')?.[6]?.innerText === 'Instead of a se
         <p>
           The main design challenge is taking what is currently paragraphs of explanation and instructions and packing them into a single test description text. Each project will involve dozens of tests like this. People will be coding the entire time, rather than switching back and forth from "reading mode" to "coding mode".
         </p>
-      </section>
+        
 --fcc-editable-region--
+      </section>
```

Comment:
```suggestion
      </section>
--fcc-editable-region--
```

I think the closing section tag should be included in the region because the opening one is.